src/base/bittorrent/torrentinfo.h

```diff
@@ -99,6 +98,7 @@ namespace BitTorrent
         void setContentLayout(TorrentContentLayout layout);
 
         std::shared_ptr<lt::torrent_info> nativeInfo() const;
+        QVector<lt::file_index_t> nativeIndexes() const;
```

Comment:
Unfortunately, it also pollutes "the outside world" with some internal things. But it is required in `torrentimpl.cpp` and `session.cpp`. Of course, I can recreate such a mapping where it is required. Or I need to split `TorrentInfo` into two classes (like `Torrent`/`TorrentImpl`). But I would leave this changes for another time, since it looks like a more general improvement than just part of this PR (`TorrentInfo `still provides `nativeInfo` which is also part of internals).